packages/client/src/components/filter-by-school-dialog.vue

```diff
@@ -46,7 +46,10 @@
             selectedSchoolCourseIds.length === 0 ? t('general.all') : undefined
           "
           :label="t('book.filters.schoolFilter.fields.course')"
-          :option-label="({ grade, section }) => `${grade}-${section}`"
+          :option-label="
+            ({ grade, section, school }: SchoolCourseFragment) =>
+              `${grade}-${section} - ${school.name}`
```

Comment:
Suggestion.
Let's render the thing like this:
`${grade}${section}` when the multiple option is not active.
When the multiple option is active, then is is rendered like `${grade}${section} - ${school.name}`